Makefile

```diff
@@ -0,0 +1,7 @@
+crx:
+	./makecrx.sh
+
+xpi:
```

Comment:
@Hainish how do you feel about Makefiles? I like to use them for organizing build scripts, and other house keeping stuff. But I probably shouldn't add this here unless we start filling it out more in the future.

Reply:
Makefiles should not be used in favor of `package.json` or at least bare .sh files. We should also avoid tying the build process to any specific OS.

Reply:
We've had a project-level Makefile in the past.  I removed it.  It fell into disuse, and since no one payed attention to it, it quickly become stale when things changed.  I'd rather avoid one altogether.